editor/action_map_editor.cpp

```diff
@@ -35,6 +35,100 @@
 #include "editor/editor_scale.h"
```

Comment:
Might be time to split this file up. It contains 3 classes now. I would do it in another pr tho.